test/sql/generated_columns/virtual/rename.test

```diff
@@ -0,0 +1,25 @@
+# name: test/sql/generated_columns/virtual/rename.test
+# description: Test renaming a generated column
+# group: [virtual]
+
+statement ok
+CREATE TABLE unit (price INTEGER, amount_sold INTEGER);
+
+statement ok
+INSERT INTO unit VALUES (5,4)
+
+statement ok
+ALTER TABLE unit ADD COLUMN total_profit INTEGER GENERATED ALWAYS AS(price * amount_sold) VIRTUAL;
```

Comment:
If you do `ADD COLUMN` and there are values that do not work, what happens? Maybe we should disallow add column for now?

Reply:
Yea that's probably best, those values dont get checked, so the error wont be detected until SELECT is ran